friendsofsymfony/user-bundle/2.1/src/Entity/User.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use FOS\UserBundle\Model\User as BaseUser;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="fos_user")
+ */
+class User extends BaseUser
+{
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\Id
```

Comment:
Can you switch order with line above please

Reply:
This entity is maybe going to disappear because it's a doctrine orm entity